website/docs/docs/fusion/fusion-readiness.md

```diff
@@ -0,0 +1,59 @@
+
+### 1. Upgrade to the latest dbt version
+
+The `Latest` [release track](/docs/dbt-versions/cloud-release-tracks) has all of the most recent features to help you prepare for <Constant name="fusion" />.
+
+- [ ] Make sure all your projects are on the `Latest` release track across all deployment environments and jobs. This will ensure the simplest, most predictable experience by allowing you to pre-validate that your project doesn't rely on deprecated behaviors. 
+
+### 2. Resolve all deprecation warnings
+
+You must resolve deprecations while your projects are on a <Constant name="core" /> release track, as they result in warnings that will become errors once you upgrade to <Constant name="fusion" />. The autofix tool can automatically resolve many deprecations (such as moving arbitrary configs into the meta dictionary). Start a new branch to begin resolving deprecation warnings using one of the following methods:
+
+- [ ] **Run autofix in the dbt platform:** You can address deprecation warnings using the [autofix tool in the Studio IDE](/docs/cloud/dbt-cloud-ide/autofix-deprecations). You can run the autofix tool on the `Compatible` or `Latest` release track.
+- [ ] **Run autofix locally:** Use the [VS Code extension](/docs/about-dbt-extension). The extension has a built-in ["Getting Started" workflow](/docs/install-dbt-extension#getting-started) that will debug your dbt project in the VS Code or Cursor IDE and execute the autofix tool. This has the added benefit of installing <Constant name="fusion" /> to your computer so you can begin testing locally before implementing in your <Constant name="dbt_platform" /> account.
+- [ ] **Run autofix locally (without the extension):** Visit the autofix [GitHub repo](https://github.com/dbt-labs/dbt-autofix) to run the tool locally if you're not using VS Code or Cursor. This will only run the tool, it will not install <Constant name="fusion" />.
+
+### 3. Validate and upgrade your dbt packages
+
+The most commonly used dbt Labs managed packages (such as `dbt_utils` and `dbt_project_evaluator`) are already compatible with <Constant name="fusion" />, as are a large number of external and community packages. We list many known supported packages [here](https://docs.getdbt.com/docs/fusion/supported-features#package-support), but more exist. 
+
+- [ ] Make sure that all of your packages are upgraded to the most recent version, which for many of them contains enhancements to support <Constant name="fusion" />. 
+- [ ] Check with your package repositories to be sure. If a package you use is not yet compatible, we recommend opening an issue with the maintainer, making the contribution yourself, or removing the package temporarily before you upgrade.
+
+### 4. Check for known Fusion limitations
+
+Your project may implement features that <Constant name="fusion" /> currently doesn't support or [limits](/docs/fusion/supported-features#limitations). 
```

Comment:
This might be just me but does it sound better flipped?
```suggestion
Your project may implement features that <Constant name="fusion" /> currently [limits](/docs/fusion/supported-features#limitations) or doesn't support. 
```